src/components/Jobs/components/RunningJobs/RunningJobs.js

```diff
@@ -5,21 +5,21 @@ import { get } from 'lodash';
 import JobsList from '../JobsList';
 import sortRunningJobs from './sortRunningJobs';
 import { PARSING_IN_PROGRESS } from '../../jobStatuses';
-import { DataFetcherContext } from '../../../DataFetcher/DataFetcherContext';
+import { DataFetcherContext } from '../../../DataFetcher';
 
 class RunningJobs extends PureComponent {
   static contextType = DataFetcherContext;
 
   prepareJobsData() {
     const jobStatuses = [PARSING_IN_PROGRESS]; // TODO: could be changed on backend
-    const jobs = get(this.context, ['jobs', 'itemsObject', 'jobExecutionDtos'], [])
+    const jobs = get(this.context, ['jobs', 'jobExecutionDtos'], [])
       .filter(({ status }) => jobStatuses.includes(status));
 
     return sortRunningJobs(jobs);
   }
 
   render() {
-    const hasLoaded = get(this.context, ['jobs', 'hasLoaded'], false);
+    const { hasLoaded = false } = this.context;
```

Comment:
Is default value needed? Why it cannot be retrieved right away from context as is?